llvmDebugInfoC/src/scripts/konan_lldb.py

```diff
@@ -241,7 +262,10 @@ def __init__(self, valobj, tip):
         else:
             log("TIP: {:#x} HIT".format(tip))
         self._children = SYNTHETIC_OBJECT_LAYOUT_CACHE[tip]
-        self._values = [self._read_value(index) for index in range(self._children_count)]
+        if VISUAL_DEBUGGER:
```

Comment:
Lazy loading values. This *may* not be very useful, but was ported from earlier version.